modules/thermal_hydraulics/joss_paper.md

```diff
@@ -72,15 +72,15 @@ used in nuclear microreactors.
 ## `Components` system
 
 The `Components` system allows users to add "components", which are very flexible
-in their use, but in general represent "pieces" of a simulation, which may couple
+in their use, but in general represent "pieces" of a simulation, which may be coupled
 together. Common uses for components include adding meshes (1D, 2D, or 3D), variables,
 equations, and output. Components provide a higher level syntax that
 hides lower level MOOSE objects such as `Kernel`s, `BoundaryCondition`s, etc. While
 `Action`s can also be used to create a higher level syntax, components provide much more
 convenience, particularly when multiple components interact.
 
 Usually components represent some physical component in a system
-such as a 1D pipe, a 2D wall/structure, or a 0D junction; however, they can also be abstract,
+such as a pipe, a solid body, or a junction; however, they can also be abstract,
```

Comment:
This is much better. But maybe also change the line above to something like: "Usually components represent some physical part in a system", so that it is not "component represents a component"

Reply:
Ok, done